### PR TITLE
Add ImperialQuart to Volume

### PR DESCRIPTION
**Pull request by Eternali**

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/VolumeTests.cs [hunk 1]
```diff
@@ -106,6 +106,11 @@ public class VolumeTests : VolumeTestsBase
 
         protected override double DecalitersInOneCubicMeter => 1e2;
 
+        /// <summary>
+        /// https://www.legislation.gov.uk/uksi/1995/1804/made
+        /// </summary>
+        protected override double ImperialQuartsInOneCubicMeter => 879.876993196;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeTests.cs:112]: Verified.

Weird, wolfram normally is very precise, but seems to round here.
https://www.wolframalpha.com/input?i=1+cubic+meter+in+imperial+quarts

File: Common/UnitDefinitions/Volume.json [hunk 2]
```diff
@@ -461,6 +461,18 @@
         }
       ]
     },
+    {
+      "SingularName": "ImperialQuart",
+      "PluralName": "ImperialQuarts",
+      "FromUnitToBaseFunc": "{x} * 1.1365225e-3",
+      "FromBaseToUnitFunc": "{x} / 1.1365225e-3",
```
Comment (angularsen) [inline Common/UnitDefinitions/Volume.json:468]: Verified 
> The imperial quart, which is used for both liquid and dry capacity, is equal to one quarter of an imperial gallon, or exactly 1.1365225 liters. 

https://en.wikipedia.org/wiki/Quart

**Review by angularsen — APPROVED** [review]

**Comment by angularsen** [comment]
Looks perfect, thanks. 
We could also add US quarts, if you are up for it? Merging this as-is anyway.

**Comment by Eternali** [comment]
Thank you for the quick response!
Looks like U.S. quarts is already part of the Volume class which is why I only needed to add the Imperial quart.

**Comment by angularsen** [comment]
Lol, it was literally right next to it and I still missed it 😅

Nuget should be out shortly.
[Release UnitsNet/5.17.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F5.17.0)

**Comment by Eternali** [comment]
No problem lol, glad I wasn't missing anything, thanks for the release!

**Pull request merged**